runtime/doc/editing.txt

```diff
@@ -1236,10 +1243,18 @@ present in 'cpoptions' and "!" is not used in the command.
 							*:chd* *:chdir*
 :chd[ir][!] [path]	Same as |:cd|.
 
+							*:tc* *:tcd*
+:tc[d][!] {path}        Like |:cd|, but set the current directory for the
+                        current tab. The current directory for other tabs or
+                        the local directory of windows in this tab is not
+                        changed, except for the current window.
+
+:tchdir[!] {path}	Same as |:tcd|.
```

Comment:
If I am not mistaking, `tchdir` can be abbreviated as `tch`. Just copy from `:lchdir` documentation.